docs/hazmat/primitives/asymmetric/ec.rst

```diff
@@ -230,6 +230,13 @@ All named curves are providers of :class:`EllipticCurve`.
     SECG curve ``sect283k1``. Also called NIST K-283.
 
 
+.. class:: SECT239K1
+
+    .. versionadded:: 1.2
+
+    SECG curve ``sect239k1``.
```

Comment:
Yeah let's drop them for now. The reason we don't support them is simply that we don't have test vectors to confirm the underlying implementation. Without test vectors we don't want them to be listed as supported curves at this time.